frontend/src/metabase/nav/components/AppBar/AppBarLarge.tsx

```diff
@@ -35,26 +36,31 @@ const AppBarLarge = ({
   isNavBarVisible,
   isSearchVisible,
   isNewButtonVisible,
+  isBreadcrumbsVisible,
   isProfileLinkVisible,
   isCollectionPathVisible,
   isQuestionLineageVisible,
   onToggleNavbar,
   onLogout,
 }: AppBarLargeProps): JSX.Element => {
+  isNavBarOpen = isNavBarOpen && isNavBarVisible;
+
   return (
     <AppBarRoot isNavBarOpen={isNavBarOpen}>
       <AppBarLeftContainer isNavBarVisible={isNavBarVisible}>
-        <AppBarLogo
-          isNavBarOpen={isNavBarOpen}
-          isToggleVisible={isNavBarVisible}
-          onToggleClick={onToggleNavbar}
-        />
+        {isNavBarVisible && (
```

Comment:
Didn't get it why this is changed. This breaks the case with just a logo that is not a button to open the sidebar.

Reply:
I thought we don't want to see the Metabase logo 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/1937582/205935330-716e1b86-1457-4332-ab09-7ce3b939cb93.png">

But agree with you I should revert this back.